M2/Macaulay2/packages/CoincidentRootLoci.m2

```diff
@@ -3,17 +3,16 @@ if version#"VERSION" < "1.15" then error "this package requires Macaulay2 versio
 
 newPackage(
        "CoincidentRootLoci",
-	Version => "0.1.2", 
-        Date => "June 22, 2020",
+	Version => "0.1.3", 
+        Date => "January 30, 2021",
     	Headline => "coincident root loci",
-        Authors => {{Name => "Maria Chiara Brambilla", Email => "brambilla@dipmat.univpm.it"},
-                    {Name => "Giovanni Staglianò", Email => "giovannistagliano@gmail.com"}},
+        Authors => {{Name => "Giovanni Staglianò", Email => "giovannistagliano@gmail.com"}},
```

Comment:
Normally we don't remove co-author names, even if we delete their code, because we value their previous contribution.

Reply:
This is nice of you. I moved M. C. Brambilla from authors to contributors because she is not a co-author of this package but a co-author of the paper [here](https://www.sciencedirect.com/science/article/abs/pii/S0024379518303720).
I originally considered this package as a kind of supplemental material for that paper, but this is not the case. The paper has already been published and contains no supplementary material. The documentation starts with "This package accompanies the paper...", but actually it just contains functions (such as realrank), which are useful for verifying the correctness of some claims in the paper. However, it's not a big deal for me to keep the author list as it was.

Reply:
Leave it as is, and thank you for your explanation.